### PR TITLE
Add Beatmap Spotlights Summer 2022 newspost & wiki articles

### PR DESCRIPTION
**Pull request by venix12**
:tada:

- [x] open lobbies before merge

### CODE REVIEW

File: wiki/Beatmap_Spotlights/Seasons/2022_Summer/en.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+| Playlist lobby: Playlist C | Oct 4 - Oct 11, 2022 |
+| Season conclusion | Oct 11, 2022 |
+
+## Weekly multiplayer lobbies
+
+As of now, joining the weekly multiplayer lobbies requires participants to download and install [osu!(lazer)](https://github.com/ppy/osu/releases).
+
+1. Every 2 weeks of a season, a single playlist lobby is opened. Everyone can participate for the entire week.
+   - After the first wave of lobbies, playlists will be opened again on a weekly basis.
+2. Each playlist lobby provides a pre-defined playlist of beatmaps.
```
Comment (Blooshing) [inline wiki/Beatmap_Spotlights/Seasons/2022_Summer/en.md:29]:
```suggestion
2. Each playlist lobby provides a predetermined playlist of beatmaps.
```

File: wiki/Beatmap_Spotlights/Seasons/2022_Summer/en.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+   - Each playlist contains an even distribution of Hard, Insane and Expert beatmaps.
```
Comment (Blooshing) [inline wiki/Beatmap_Spotlights/Seasons/2022_Summer/en.md:30]:
```suggestion
   - Each playlist contains an even distribution of Hard, Insane, and Expert beatmaps.
```

**Review by Blooshing — COMMENTED** [review]
oxford comma gang

File: wiki/Beatmap_Spotlights/Seasons/en.md [hunk 3]
```diff
@@ -13,3 +13,4 @@ tags:
 - [Spring 2021](2021_Spring)
 - [Winter 2022](2022_Winter)
 - [Spring 2022](2022_Spring)
+- [Summer_2022](2022_Summer)
```
Comment (ZeroPyrozen) [inline wiki/Beatmap_Spotlights/Seasons/en.md:16]:
```suggestion
- [Summer 2022](2022_Summer)
```

**Comment by Ephemeralis** [comment]
A few banner images seem to be missing. @venix12

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**